src/libraries/System.Runtime/tests/System/DelegateTests.cs

```diff
@@ -328,6 +330,32 @@ public static void DynamicInvoke_DefaultParameter_DateTimeParameterWithMissingVa
                 (DateTime)(new DateTimeWithDefaultValueAttribute(DateTimeMethod)).DynamicInvoke(new object[] { Type.Missing }));
         }
 
+        // CoreCLR prioritizes DateTimeConstant over CustomConstant attribute
```

Comment:
```suggestion
        // Verify that DateTimeConstantAttribute is prioritized over CustomConstantAttribute
```
This behavior is not CoreCLR specific after your changes.